README.md

```diff
@@ -0,0 +1,19 @@
+# ink-wrapper
+
+## CI
+
+You should be able to run the CI processes locally with https://github.com/nektos/act:
```

Comment:
Personally, I like to create a Makefile with the commands required to build docker, start it and run the test. They can be packaged together into `make run-ci` and called from the CI as well. That way there would be a single point where things can change (Makefile) and they don't require installation of external tool (like this `act` here).

Reply:
Let me do that in the next PR - I'll just cut up the `test.sh` file into pieces in a `Makefile`. I think this is slightly different from running the whole CI with `act` - the CI will still do `docker run ... make run-ci`, but locally you might want to run against a node run on the host, etc.

Reply:
OK.